recipes/signac/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "signac" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://bitbucket.org/glotzer/signac
```

Comment:
A pull request to the feedstock would be great. Also @jakirkham thanks for catching this.